xpallets/transaction-fee/src/types.rs

```diff
@@ -41,10 +42,10 @@ pub struct InclusionFee<Balance> {
 #[derive(Encode, Decode, Clone, Eq, PartialEq, RuntimeDebug)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "std", serde(rename_all = "camelCase"))]
-pub struct FeeDetails<Balance> {
-    pub inclusion_fee: Option<InclusionFee<Balance>>,
+pub struct FeeDetails<Balance: Default> {
     /// Some calls might be charged extra fee besides the essential `inclusion_fee`.
+    /// use the struct Feedetails in substrate instead of the previous protery inclusion_fee and tip
+    pub partial_details: pallet_transaction_payment::FeeDetails<Balance>,
```

Comment:
Make the name a little shorter?

```suggestion
    pub base: pallet_transaction_payment::FeeDetails<Balance>,
```